README.md

```diff
@@ -0,0 +1,400 @@
+# Sweater Weather Express
+
+### Introduction
+
+Sweater Weather Express is a backend application written in Express.js designed to supply formatted JSON API endpoints. The application is designed to a provided specification for integration with a potential future frontend, and is expected to manage a database of users who will be authenticated using a unique API key, make forecast requests for city queries on behalf of authenticated users, and manage favorited locations on behalf of authenticated users.
+
+### How to use
+
+The following dependencies are required:
+- Node.js
+- Express.js
+- PostgreSQL
+
+##### Initial Setup
+
+``` bash
+npm install # Install dependencies
+npx sequelize db:create # Create PostgreSQL Database
+npx sequelize db:migrate # Run migrations for database setup
+```
```

Comment:
Does a user need to install anything else (dotenv, fetch, etc)? Maybe not...

Reply:
I think they should be covered by the npm install for all the remaining packages. I could add a suggestion to use nodemon for the development server since that wouldn't be covered by the package file. Do you think that would be beneficial?